src/pages/journal/journal-force-check-modal/journal-force-check-modal.module.ts

```diff
@@ -0,0 +1,18 @@
+import { JournalForceCheckModal } from './journal-force-check-modal';
+import { NgModule } from '@angular/core';
+import { IonicPageModule } from 'ionic-angular';
+import { TestReportModalModule } from '../../test-report/components/test-report-modal/test-report-modal.module';
+
+@NgModule({
+  declarations: [
```

Comment:
this should be nested in components like the other modals in the app so 

` src/pages/journal/components/journal-force-check-modal/xxx